### ISSUE TITLE
Missing bot messages

### ISSUE THREAD

**Issue by twouters** [Sorunome/mx-puppet-slack#34]
Haven't put much thought in this myself yet but some bot messages aren't getting through:

```json
{
  "bot_id":"B55XXXXXX",
  "suppress_notification":false,
  "type":"message",
  "text":"",
  "user":"U55XXXXXX",
  "team":"T04XXXXXX",
  "bot_profile":{
    "id":"B55XXXXXX",
    "deleted":false,
    "name":"bot",
    "updated":1493364237,
    "app_id":"A0FXXXXXX",
    "icons":{
      "image_36":"https:\/\/a.slack-edge.com\/XXXXX\/img\/services\/bots_36.png",
      "image_48":"https:\/\/a.slack-edge.com\/XXXXX\/img\/plugins\/bot\/service_48.png",
      "image_72":"https:\/\/a.slack-edge.com\/XXXXXX\/img\/services\/bots_72.png"
    },
    "team_id":"T04XXXXXX"
  },
  "attachments":[{
    "text":"Sometext contents",
    "id":1,
    "fallback":"Sometext contents"
  }],
  "source_team":"T04XXXXXX",
  "user_team":"T04XXXXXX",
  "channel":"G2ZXXXXXX",
  "event_ts":"1581582105.469000",
  "ts":"1581582105.469000"
}
```

Just a detail: this bot message is a response to a message that I've sent and it arrives pretty fast.
My message got detected by the bridge at `Feb-13 09:21:44.834 [MatrixEventHandler] info: Got new message in <room> from @<user>`

timestamps of messages according to slack:  
my message: 1581582104.468900
bot message: 1581582105.469000

I don't have an exact timestamp of when the messages got back to the bridge but it looks like they arrive pretty much at the same time.

**Comment by twouters** [comment]
Found more logs:

```
Feb-13 09:21:44.997 [SlackPuppet:slack] verbose: Got new message event
Feb-13 09:21:44.998 [SlackPuppet:slack] verbose: Received message. subtype=undefined files=0
Feb-13 09:21:45.131 [SlackPuppet:slack] verbose: Got new message event
Feb-13 09:21:45.132 [SlackPuppet:slack] verbose: Received message. subtype=undefined files=0
```

**Comment by Sorunome** [comment]
Turns out the issue was that soru only checked for an empty message if the text was empty, and ignored if it still had attachments.

Should be fixed in https://github.com/Sorunome/mx-puppet-slack/commit/f8048dbf0ac6cdc8181be521cbb596e37845b154

**Comment by twouters** [comment]
@Sorunome this issue popped back up, not sure if it's the same issue or not.
I have not taken the time to check the code yet.

**Comment by Sorunome** [comment]
Accidnetal regression, @matejdro kindly fixed it in #44
